omisego-sdk/src/main/java/co/omisego/omisego/custom/CallbackManager.kt

```diff
@@ -0,0 +1,31 @@
+package co.omisego.omisego.custom
+
+/**
+ * OmiseGO
+ *
+ * Created by Phuchit Sirimongkolsathien on 5/3/2018 AD.
+ * Copyright © 2017-2018 OmiseGO. All rights reserved.
+ */
+
+import com.google.gson.JsonElement
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import java.lang.reflect.Type
+import co.omisego.omisego.custom.Callback as OMGCallback
+
+internal class CallbackManager<out T>(private val serializer: Serializer, private val type: Type) {
```

Comment:
Just detail here, but `Manager` is not a very clear name, you can't guess what it's doing.
It's an internal class so not a big deal, but it seems to be mapper (or even wrapper?) from retrofit's `Callback` to OMG's `Callback`

Reply:
Remove this class